src/bin/menu.rs

```diff
@@ -69,9 +69,11 @@ where
     }
 
     pub fn answer(&self, answer: &Answer) -> &K {
-        let index = answer.as_list_item().unwrap().index;
+        // PANIC: Okay to panic here since its called by iteractive
```

Comment:
Small nit: `iteractive` -> `interactive`. 

I assume the reason for panicking here being okay isn't per se that you're in interactive mode, but also because it should logically not reach a point where this can happen.